api/fixtures/example.go

```diff
@@ -167,7 +167,6 @@ aws_secret_access_key = %s
 			Release: hyperv1.Release{
 				Image: o.ReleaseImage,
 			},
-			InitialComputeReplicas: o.NodePoolReplicas,
```

Comment:
There's still the `--node-pool-replicas` flag for `create cluster`: https://github.com/openshift/hypershift/blob/main/cmd/cluster/create.go#L228

We'll need to deal with it one way or another. Some possibilities:

1. Remove `--node-pool-replicas` from the `create cluster` command entirely and update the README to be more clear about the two-stage workflow
2. Keep the flag and refactor the fixture setup to render a NodePool based on `--node-pool-replicas`

Thoughts?

Reply:
I think option 2 is best.  Reworking...